.cargo/config.toml

```diff
@@ -2,3 +2,8 @@
 rustflags = [
   "-C", "link-arg=-Wl,-E"
 ]
+
+[target.aarch64-unknown-linux-gnu]
+rustflags = [
+  "-C", "link-arg=-Wl,-E"
```

Comment:
Should we add them as a flag always? (not just fofr x86_64 or aarch64, but for all architectures)

Reply:
That would be great, but the flag only works on Linux. I don't think this file allows globs or regexes?

Reply:
Done.